### PR TITLE
Fix typos in comments and docs

### PR DESCRIPTION
**Pull request by upbooo**
## Proposed Changes

There were typos in comments and docs so I decided to fix it.

**Release Note**

```release-note
NONE
```

### CODE REVIEW

**Comment by upbooo** [comment]
@googlebot I signed it!

File: internal/verification/phaverify_test.go [hunk 1]
```diff
@@ -14,7 +14,7 @@
 
 // Package verification provides the ability to verify the diagnosis certificates
 // (JWTs) coming from public health authorities that are responsible for verifying
-// diagnosis pin codes and ceritfying the TEKs.
+// diagnosis pin codes and verifying the TEKs.
```
Comment (mikehelmick) [inline internal/verification/phaverify_test.go:17]: This is not a typo

File: pkg/api/v1alpha1/verification_types.go [hunk 2]
```diff
@@ -53,15 +53,15 @@ const (
 )
 
 // TransmissionRiskVector is an additional set of claims that can be
-// included in the verification certificatin for a diagnosis as received
+// included in the verification / certification for a diagnosis as received
```
Comment (mikehelmick) [inline pkg/api/v1alpha1/verification_types.go:56]: Should be certificate

**Review by mikehelmick — COMMENTED** [review]
/ok-to-test

Thanks!

File: internal/verification/phaverify_test.go (hunk 1 above)
Comment (sethvargo) [reply internal/verification/phaverify_test.go:17]: It is a typo `ceritfying -> certifying`. But `verifying` is not correct.

File: internal/verification/phaverify.go [hunk 3]
```diff
@@ -14,7 +14,7 @@
 
 // Package verification provides the ability to verify the diagnosis certificates
 // (JWTs) coming from public health authorities that are responsible for verifying
-// diagnosis pin codes and ceritfying the TEKs.
+// diagnosis pin codes and verifying the TEKs.
```
Comment (sethvargo) [inline internal/verification/phaverify.go]:
```suggestion
// diagnosis pin codes and certifying the TEKs.
```

File: internal/verification/phaverify_test.go (hunk 1 above)
Comment (sethvargo) [inline internal/verification/phaverify_test.go:17]:
```suggestion
// diagnosis pin codes and certifying the TEKs.
```

File: pkg/api/v1alpha1/verification_types.go (hunk 2 above)
Comment (sethvargo) [inline pkg/api/v1alpha1/verification_types.go]:
```suggestion
// included in the verification / certificate for a diagnosis as received
```

**Review by sethvargo — CHANGES_REQUESTED** [review]

**Comment by upbooo** [comment]
Looks good now, thanks for your guidance 👍

File: pkg/api/v1alpha1/verification_types.go [hunk 4]
```diff
@@ -53,15 +53,15 @@ const (
 )
 
 // TransmissionRiskVector is an additional set of claims that can be
-// included in the verification certificatin for a diagnosis as received
+// included in the verification / certificate for a diagnosis as received
```
Comment (mikehelmick) [inline pkg/api/v1alpha1/verification_types.go:56]:
```suggestion
// included in the verification certificate for a diagnosis as received
```

**Review by sethvargo — APPROVED** [review]
/lgtm

**Pull request merged**